README.md

```diff
@@ -4,18 +4,84 @@ This repository contains a few practice assignments on git.<br/>
 First, setup git and create a github account as explained in the slides.<br/>
 
 Execute the following steps to practice various topics covered in the intro to git session<br/>
-1) Fork the repository using the fork icon.<br/>
-2) Clone the forked repository into a directory of your choice in your local machine using the command git clone <url_of_forked_repository> <br/>
-3) Create a branch with name myFirstBranch using the command git branch <br/>
-4) Now switch to the new branch using the command git checkout myFirstBranch<br/>
-5) Add a sentence of your choice to the sample.txt file. (Ex : Hi I am <your_name>) <br/>
-6) Add a new text file named new.txt <br/>
-7) Now, use the commands git status and git diff and see what happens <br/>
-8) Use the command git add --all to stage all the changes <br/>
-9) Again use the commands git status and git diff and see what happens <br/>
-10) Now, commit the changes using git commit -m "my first commit" <br/>
-11) Switch to the main branch by using the command git checkout main <br/>
-12) Now merge the other branch with main branch using git merge myFirstBranch <br/>
-13) Now push all the changes onto your forked github repository using git push origin/main <br/>
+# 1) Forking a repository 
+   Fork the repository using the fork icon.<br/> <br/>
+   ![](images/fork.png)
+    <hr>
+# 2) Cloning a forked repository 
+   Clone the forked repository into a directory of your choice in your local machine using the command `git clone <url_of_forked_repository>` <br/> <br/>
+   ![](images/clone.png)
+    <hr>
+# 3) Branching
+   Create a branch with name myFirstBranch using the command `git branch myFirstBranch` <br/> <br/>
+   ![](images/git_branch.png)
+    <hr>
+# 4) Switching to a new branch
+   Now switch to the new branch using the command `git checkout myFirstBranch` <br/> <br/>
+   ![](images/git_checkout.png) <br/> <br/>
+   As shown in the above image, the branch gets changed from main to my first branch <br/>
+    <hr>
+   # Note : 
+   The above two steps can be performed at once by using the command `git checkout -b "myFirstBranch"`. This command creates a new branch with the name specified in the quotes and switches to that branch.
+    <hr>
+# 5) Making changes
```

Comment:
It's not very clear as to which files come in as unstaged and which files come in as untracked when created. Could you specify this too here? I don't think any files are untracked when created in the repo, they are only unstaged. Or I'm wrong?

Reply:
![image](https://github.com/Programming-Club-IITM/GitPractice/assets/161064803/b6fbe2b1-8afa-4b4e-a874-39f51b2944a7)
As per the above screenshot, the new file that we added is untracked but the changes to the old file are tracked and unstaged.

Reply:
Should I specify this under the making changes heading?

Reply:
Yes in the make changes heading. I meant is it like when you create a new file is it untracked or unstaged? And when you edit it what happens etc.

Reply:
Am not sure, but when we create a new file I think it will be untracked. When we modify an already tracked file, I think it will be an unstaged change. I wrote a bit about this in heading number 6

Reply:
Ok then find out and make it clear in part 5 itself. What happens when I create a new file, what happens when I edit it etc. Tracking and Staging is a little confusing when I read this